dvc/testing/workspace_tests.py

```diff
@@ -106,6 +106,7 @@ def test_import_file(self, tmp_dir, dvc, remote_version_aware):
         (remote_version_aware / "file").write_text("modified")
         assert dvc.status().get("file.dvc") == [
             {"changed deps": {"remote://upstream/file": "update available"}},
+            {"changed outs": {"file": "not in cache"}},
```

Comment:
`dvc status` doesn't yet work with fs-based cache, so this will have to wait a bit for https://github.com/iterative/dvc/issues/9333